### DISCUSSION TITLE
Confused about Chatbot Mode

### DISCUSSION THREAD

**Discussion by LiamPerson** [KoboldAI/KoboldAI-Client#363] [Q&A] [answered]
I want to use the Chatbot Mode https://github.com/KoboldAI/KoboldAI-Client/wiki#chatbot-mode 

Where do I write the _"story opening"_ as written in the wiki page linked above? Do I just send that as the first message, or do insert that as the memory?

![image](https://github.com/KoboldAI/KoboldAI-Client/assets/34695601/32c34135-9bb6-4bc8-b79b-073b57f7cf93)

**Comment by henk717** [comment, marked as answer]
The wiki page was written for a different UI, but on Lite you can use the edit button to get direct control and then paste or write your opening to the editor. After you turn edit mode back off it is the fancy UI again.
